sdk/monitor/Azure.Monitor.Query/src/LogsQueryClient.cs

```diff
@@ -66,14 +68,24 @@ public LogsQueryClient(Uri endpoint, TokenCredential credential) : this(endpoint
         /// <param name="endpoint">The service endpoint to use.</param>
         /// <param name="credential">The <see cref="TokenCredential"/> instance to use for authentication.</param>
         /// <param name="options">The <see cref="LogsQueryClientOptions"/> instance to use as client configuration.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public LogsQueryClient(Uri endpoint, TokenCredential credential, LogsQueryClientOptions options)
         {
             Argument.AssertNotNull(credential, nameof(credential));
             Argument.AssertNotNull(endpoint, nameof(endpoint));
 
             Endpoint = endpoint;
             options ??= new LogsQueryClientOptions();
-            var authorizationScope = $"{(string.IsNullOrEmpty(options.Audience?.ToString()) ? LogsQueryAudience.AzurePublicCloud : options.Audience)}";
+            if (string.IsNullOrEmpty(options.Audience?.ToString()))
+            {
+                options.Audience = endpoint.OriginalString;
```

Comment:
It would probably be better to avoid mutating the options instance since this was passed by the user. They shouldn't hold onto the reference but better to avoid mutating if not needed.